nservicebus/router/index.md

```diff
@@ -0,0 +1,88 @@
+
+In addition to immediate and delayed retries, the router has built-in outage detection through a *circuit breaker*. After a number of consecutive failures the circuit breaker is triggered which causes the interface to enter the *throttled mode*. In this mode the interface processes a single message at a time and pauses after each processing attempt. The interface goes back to the normal mode after the first successful processing attempt. When in the throttled mode the router does not increment the delayed retries counter to prevent messages being sent to the poison message queue due to infrastructure outages.
+
+## Topologies
+
+A router consists of multiple [NServiceBus.Raw](/nservicebus/rawmessaging/) endpoints, called *interfaces* and a *routing protocol* that controls how messages should be forwarded between them.
+
+
+### Bridge
+
+![Bridge](bridge.svg)
+
+The arrows show the path of messages sent from `Endpoint A` to `Endpoint C` and from `Endpoint D` to `Endpoint B`. The messages cross the router in the opposite direction. Each message is initially sent to the router queue (based on the routing configuration of the connector) and then forwarded to the destination queue. There is one additional *hop* compared to a direct communication of endpoints. The following snippet configures the built-in *static routing protocol* to forward messages between the router's interfaces.
+
+snippet: simple-routing
+
+
+### Multi-way routing
+
+![Multi-way](multi-way.svg)
+
+The router is not limited to only two interfaces but in case there is more than two interfaces, the routing protocol rules need to be more complex and specific. The following snippet configures the built-in *static routing protocol* to forward messages to interfaces based on the prefix of the destination endpoint's name.
+
+snippet: three-way-router
+
+NOTE: All three interfaces use the same transport type (SQL Server Transport) but may use different settings e.g. a different database instances. This way each part of the system (Sales, Shipping and Billing) can be autonomous and own its database server yet they still can exchange messages in the same way as if they were connected to a single shared instance.
+
+
+### Backplane
+
+![Backplane](backplane.svg)
+
+Two or more routers can be connected together to form a _backplane_ topology. This setup usually makes most sens for the geo-distributed systems. The following snippet configures the router hosted in the Europen part of the globally distributed system to route messages coming from outsize via the Azure Storage Queues interface directly to the local endpoints and to route messages sent by local endpoints to either East or West United States through *designated gateway* routers.
+
+NOTE: The *designated gateway* concept is not related to the NServiceBus.Gateway package. When the *designated gateway* is specified in the route, the message is forawrded to it instead of the actual destination.
+
+snippet: three-way-router
```

Comment:
This is the exact name snippet name as the one above? Is that correct?

Reply:
Nope. A copy-paste error 👍